spec/game_spec.rb

```diff
@@ -1,55 +1,57 @@
+require "csv"
 require 'spec_helper'
 
 RSpec.describe Game do
-  describe "initialize" do
-    it "exists and has attributes" do 
-      game_1 = Game.new({
-        game_id: 2012030221,
-        season: 20122013,
-        type: "Postseason",
-        date_time: "5/16/13",
-        away_team_id: 3,
-        home_team_id: 6,
-        away_goals: 2,
-        home_goals: 3,
-        venue: "Toyota Stadium",
-        venue_link: "/api/v1/venues/null"
-        })
-
-      expect(game_1).to be_a(Game)
-      expect(game_1.game_id).to eq(2012030221)
-      expect(game_1.season).to eq(20122013)
-      expect(game_1.type).to eq("Postseason")
-      expect(game_1.date_time).to eq("5/16/13")
-      expect(game_1.away_team_id).to eq(3)
-      expect(game_1.home_team_id).to eq(6)
-      expect(game_1.away_goals).to eq(2)
-      expect(game_1.home_goals).to eq(3)
-      expect(game_1.venue).to eq("Toyota Stadium")
-      expect(game_1.venue_link).to eq("/api/v1/venues/null")
+  let(:game) {Game.new(row)}
```

Comment:
During our group discussions we talked about the benefits of testing a single local variable and pulling from the csv file to show that our Class is dynamic and can work for a variety of data. Let's discuss if we want to keep this vs change with the group

Reply:
I could be wrong but I believe this is a single local variable to show our class is dynamic. We should that it can pull or read from the csv through the stat_tracker_spec. Because ultimately our game class just creates a game object, we have no code saying read or open CSV file in our game class.

`:game` in the let statement is the single local variable, I am just using a `let` block because it is performed lazily and also memorizes the value or rather creates a cache. So for performance, the `let` block only runs when you call it and only has to run once, instead of having to use a before(:each) block for example. 

 So it is kinda like writing game = Game.new(row), but doing it in a let block it improves performance and prevents problems with repeated computations or data manipulation.